packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -544,7 +545,15 @@ export class ModularChangeFamily
 				field.change = fieldKind.changeHandler.rebaser.rebase(
 					fieldChangeset,
 					tagChange(baseChangeset, context.baseRevision),
-					(node) => node,
+					(curr, base) => {
+						if (curr === undefined) {
```

Comment:
Can we refactor this block to use `curr ?? base` as the key for a single lookup? Then we would only have two cases (key defined or undefined) instead of the three we have here.